doc/classes/DisplayServer.xml

```diff
@@ -20,6 +20,12 @@
 				[b]Note:[/b] This method is only implemented on Linux.
 			</description>
 		</method>
+		<method name="clipboard_has" qualifiers="const">
+			<return type="bool" />
+			<description>
+				Returns [code]true[/code] if there is content on the the user's clipboard.
```

Comment:
```suggestion
				Returns [code]true[/code] if there is content in the user's clipboard.
```

Reply:
Ah, the typo :P

For "in", I checked [Windows](https://docs.microsoft.com/en-us/windows/win32/dataxchg/clipboard), [macOS/iOS](https://developer.apple.com/documentation/uikit/uipasteboard), and [Android](https://developer.android.com/reference/android/content/ClipboardManager#hasPrimaryClip()) APIs, and they all use "on the {clip,paste}board".

Reply:
I guess technically that's correct since it's a "board". I always imagine it as a container but I'm probably mistaken :D